### PR TITLE
Load parent assets in shop selector iframe page

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix product shops selection modal in multishop
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Install the shop with multiple shops, create a new product try to associated it with other shops, without the PR the modal content is not integrated and can't be used, with this fix it works as expected (see screenshots)
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/9497638564
| Fixed issue or discussion?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### Without fix
Capture d’écran 2024-06-13 à 11 46 58

### With fix
Capture d’écran 2024-06-13 à 11 47 09

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

Please consider opening an issue before submitting a Pull Request:
* If it's a bug fix, it helps maintainers verify that the bug is effectively due to a defect in the code, and that it hasn't been fixed already.
* It can help trigger a discussion about the best implementation path before a single line of code is written.
* It may lead the Core Product team to mark that issue as a priority, further attracting the maintainers' attention.

(Note: this is an automated message, but answering it will reach a real human)

**Review by boherm — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @jolelievre ,

Checked the multistore modal, checked for all shops and all group shops.
Checked migrated and legacy pages.
Behavior and design are as expected.

It is QA ✅

**Comment by ps-jarvis** [comment]
QA approved, well done! Message to the maintainers: do not forget to milestone it before the merge.

**Comment by jolelievre** [comment]
Thanks @florine2623

**Pull request merged**
